packages/dotcom-ui-base-styles/styles.scss

```diff
@@ -2,7 +2,7 @@ $system-code: 'page-kit-base-styles' !default;
 
 // NOTE: please also update ./src/lib/fontFaces.ts when updating the URL below. They
 // need to stay in-sync to ensure our CSS output matches the resource hints we set.
-$o-fonts-path: 'https://www.ft.com/__origami/service/build/v2/files/o-fonts-assets@1.5.0/';
+$o-fonts-path: 'https://www.ft.com/__origami/service/build/v2/files/o-fonts-assets@1.7.0/';
```

Comment:
this is kinda true, kinda a lie, because of multiple Sass entry points of ft projects